src/main/java/seedu/address/model/entry/util/EntryAutofill.java

```diff
@@ -57,12 +51,16 @@ public EntryAutofill(Title originalTitle, Description originalDescription) {
         });
     }
 
+    private boolean noTitleOrNoDescription() {
+        return originalEntry.getTitle().isEmpty() || originalEntry.getDescription().isEmpty();
```

Comment:
Calculate this just once in the constructor instead? Since this is effectively final.

Reply:
good point!